modules/claims_api/config/schemas/v2/2122.json

```diff
@@ -218,15 +219,10 @@
           "type": "string",
           "example": "I help vets LLC."
         },
-        "firstName": {
```

Comment:
I realized with removing first and last name that the form constructor needs to change. `modules/claims_api/lib/claims_api/v2/poa_pdf_constructor/organization.rb` item 16A uses the first and last name attributes